cypress/integration/dialogs/large-format-dialog.ts

```diff
@@ -0,0 +1,19 @@
+describe('Large Format Dialog', () => {
+  before(() => {
+    cy.visit('/dialog-large-format')
```

Comment:
does prettier get run on test files? it's missing a semicolon

Reply:
Nope... but it should.